src/main/resources/application.properties

```diff
@@ -108,8 +108,8 @@ cron-scheduler-unblock=0 0 0 * * ? *
 
 spring.mail.host=smtp.gmail.com
 spring.mail.port=587
-spring.mail.username=<Enter mail username>
-spring.mail.password=<Enter mail password>
+spring.mail.username=shekhar.shubham014@gmail.com
```

Comment:
@swetatech24 You should not put username and password in properties file. Replace this with the place holder